### ISSUE TITLE
Migrate away from PHP

### ISSUE THREAD

**Issue by SevereOverfl0w** [reignite/burnout.io#6]
This site would work great as a static site. Using a static site generator (Like Jekyll), it would be possible to have github actually auto-build your site and host it for you.

Jekyll will auto-compile your markdown, and other such goodies. Definitely worth consideration.

**Comment by balupton** [comment]
@jnettome seems progress stalled?

I'm happy to do this too. I can see if [Bevry](http://bevry.me) wants to add their resources to this too if you'd like. We currently use [DocPad](http://docpad.org) for several things, such as staticsitegenerators.net which pulls in a YAML file and regenerates on changes to the YAML file, as well as for the docpad website it pulling in a markdown repo and regenerating on changes as well. It's a good option if a data file approach is done for resources.

**Comment by nvtkaszpir** [comment]
Wow, I'm surpirsed this was not done as static webpage. You can rewrite it in restructured text, run sphinx build in python and generate a static webpage hosted for free on Read The Docs (RTD) - https://read-the-docs.readthedocs.org/en/latest/getting_started.html to be automatically downloaded from github, compiled by RTD and then published on their site.

**Comment by AaronSachs** [comment]
@nvtkaszpir , yeah, it looks like something that could be pretty quickly done. I've been thinking about taking this up. I'll take a gander at it tonight.

**Comment by AaronSachs** [comment]
@bemosior @nvtkaszpir So, I've got a semi-working version of the site up at RTD:  http://burnoutio-test.readthedocs.org/en/latest/index.html . I've not been able to get to the resources yet, but am working on it. I've kind of followed what [Ops School](https://ops-school.readthedocs.org/en/latest/index.html#) is doing with their stuff, and have got a passing build on RTD & Travis. I've followed suit with what I've seen there, and used rst for the formatting/content. We could just as well keep the markdown as Michal suggested. So, thoughts?
